scvid/module/from_cli.py

```diff
@@ -0,0 +1,96 @@
+# Copyright Contributors to the Cellarium project.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import math
+
+import torch
+
+from scvid.transforms import ZScoreLog1pNormalize
+
+from ._probabilisticpca import ProbabilisticPCA
+from .onepass_mean_var_std import OnePassMeanVarStd
+
+
+class OnePassMeanVarStdFromCli(OnePassMeanVarStd):
+    """
+    Preset default values for the LightningCLI.
+
+    Args:
+        g_genes: Number of genes.
+        target_count: Target gene epxression count. Default: ``10_000``.
+    """
+
+    def __init__(self, g_genes, target_count: int = 10_000) -> None:
```

Comment:
We must think of a mechanism to synchronize the `target_count` used here and the target count used by PCA modules.  At the very least, there must be an assert statement to ensure that mean and std that are precomputed here could only be used by a PCA module that is instantiated with the same `target_count`.

Reply:
I reverted back `ModuleCheckpoint` to save the model itself and not the `state_dict`. This way it saves `target_count`. Then we can use an assert statement for `target_count` when `ProbabilisticPCAFromCLI` is instantiated.

Another consideration here is that when the model is deployed we need to be able to load it and perform embedding. After #62 is merged the model (`IncrementalPCA`) can be saved with `ModuleCheckpoint` and then @fedorgrab should be able to load it and perform embedding with it.